client/client.go

```diff
@@ -134,9 +95,12 @@ func (c *Client) createConsumerChannel() {
 func New(url string, args ...interface{}) *Client {
```

Comment:
Not a part of this fix maybe, but we should revert my bad idéa to use unspecified amount of arguments. If we can make the client not connect when calling New we could chain things like setting dialer, TLS etcetera.

Reply:
Yep, I agree about that.